learn/src/main/java/org/navgurukul/learn/ui/learn/viewholder/CodeCourseViewHolder.kt

```diff
@@ -52,10 +52,7 @@ class CodeCourseViewHolder(itemView: View) :
             }
         }
 
-        codeBody.text = HtmlCompat.fromHtml(
-            item.value
-                ?: "", HtmlCompat.FROM_HTML_MODE_COMPACT
-        )
+        codeBody.text = item.value?.replace("<br>", "\n")?.replace("&emsp;", " ")
```

Comment:
since '\<br>' , \n etc. are used in many other pplaces as well for similar logic, then let's create a constant string to ensure no redundant code is there and it becomes easy to modify at single place.

Reply:
So can I create an object containing these strings and use them in different files for <br> and \n, as it is used in several part of project?

Reply:
Please check in latest commit https://github.com/navgurukul/soni-sori/pull/258/commits/33158e7d6de12b12834b3d3035945ea06a147bb6